doc/api/next_api_changes/behavior/XXXXX-AL.rst

```diff
@@ -0,0 +1,12 @@
+dark_background and fivethirtyeight styles no longer set ``savefig.facecolor`` and ``savefig.edgecolor``
```

Comment:
This filename is not updated with the PR number.

Reply:
oops, fixed